thingif/src/main/java/com/kii/thingif/internal/gson/AggregationAdapter.java

```diff
@@ -0,0 +1,23 @@
+package com.kii.thingif.internal.gson;
+
+import com.google.gson.JsonElement;
+import com.google.gson.JsonObject;
+import com.google.gson.JsonSerializationContext;
+import com.google.gson.JsonSerializer;
+import com.kii.thingif.query.Aggregation;
+
+import java.lang.reflect.Type;
+
+public class AggregationAdapter implements JsonSerializer<Aggregation> {

```

Comment:
I think we don't need AggregationAdapter. Simple to use `gson.toJson(aggregation)` is enough.